tools/gluten-te/centos/defaults.conf

```diff
@@ -21,17 +21,25 @@ DEFAULT_DEBUG_BUILD=OFF
 DEFAULT_BUILD_BACKEND_TYPE=velox
 
 # HTTP proxy
+# If http proxy is http://XXX.com:123, write it as
+# DEFAULT_HTTP_PROXY_HOST=xxx.com
+# DEFAULT_HTTP_PROXY_PORT=123
+# do not write http://
 DEFAULT_HTTP_PROXY_HOST=
 DEFAULT_HTTP_PROXY_PORT=
 
 # If on, use maven mirror settings for PRC's network environment
-DEFAULT_USE_ALI_MAVEN_MIRROR=OFF
+# DEFAULT_USE_ALI_MAVEN_MIRROR=OFF
+# for 10.0.0.25 test 
+DEFAULT_USE_ALI_MAVEN_MIRROR=ON
 
 # Base operator system image used in build scripts.
-DEFAULT_OS_IMAGE=centos:8
+# DEFAULT_OS_IMAGE_NAME=ubuntu or centos
+DEFAULT_OS_IMAGE_NAME=
 
 # Version ID of os image
-DEFAULT_OS_VERSION=8
+# DEFAULT_OS_IMAGE_TAG=20.04 or 8
+DEFAULT_OS_IMAGE_TAG=
```

Comment:
Minor: Why removing the default values? We'd better to let user having a default settings to run the scripts. For centos scripts it's OK to have a default base image `centos:8` IIUC.